repository/CargoPackageManager/CGOOngoingPackageUnit.class.st

```diff
@@ -94,6 +94,11 @@ CGOOngoingPackageUnit >> isUnit [
 	^ true
 ]
 
+{ #category : #loading }
+CGOOngoingPackageUnit >> load [
```

Comment:
This one should be called `basicLoad` or `doLoad`, because this one is internal. External users do not need to know about solving, that's a Cargo internal detail. To me because of that the public API should be `load` and this method should be private to cargo.